test/custom-matter-xml.test.js

```diff
@@ -312,6 +312,10 @@ test(
     expect(sdkExt).toContain(
       "/ command: 0x0006 / 0xFFF201 => SampleMfgSpecificToggleWithTransition2, test extension: ''"
     )
+    // checking the following in relation to bug ZAPP-1576
```

Comment:
Add description on what this test is testing and do not put internal issue IDs unless they are open issues for the future.